includes/polling/os/ies5000.inc.php

```diff
@@ -0,0 +1,10 @@
+<?php
+# ies 5106
+if ($device['sysObjectID'] == '.1.3.6.1.4.1.890.1.5.13.10') {
+    $r_temp = snmp_get_multi_oid($device, ['.1.3.6.1.4.1.890.1.5.13.5.6.2.1.3.0', '.1.3.6.1.4.1.8886.6.1.1.1.2.0', '.1.3.6.1.4.1.8886.6.1.1.1.14.0'], '-OQUn');
+    $serial   = $r_temp['.1.3.6.1.4.1.890.1.5.13.5.6.2.1.3.0'];
```

Comment:
Hello,
You poll a few OIDs but only use one of them ? If you don't need the others, please avoid polling them. 
Thx

Reply:
fixed, thx for hint.